### ISSUE TITLE
Problème code calcul TVA

### ISSUE THREAD

**Issue by ghost** [seinecle/codapps#89]
Même en recopiant le tutoriel, je n'arrive pas à obtenir le prix avec TVA !
![probleme ajout tva](https://user-images.githubusercontent.com/35698106/37252856-1e096b44-2529-11e8-98e3-8ff5a847a634.PNG)

**Comment by seinecle** [comment]
Good point!

The line `price = price * 1.20;` in the method addFrenchVAT is incorrect.

The program does not know if 1.20 is a float or a double, since we did not tell. So it chooses double, which creates a bug: multiplying the price (a Float) with 1.20 (a double) is not possible.

The solution: `price = price * 1.20f;` (note I added a `f` at the end of `1.20`)

I will correct the course material this afternoon. Thank you for spotting this error!
